examples/gno.land/p/demo/gnofundme/gnofundme.gno

```diff
@@ -0,0 +1,124 @@
+package gnofundme
+
+import (
+	"std"
+	"strconv"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/avl"
+)
+
+type Campaign struct {
+	title        string
+	description  string
+	goal         uint
+	current      uint
+	begin        time.Time
+	deadline     time.Time
+	owner        *User
+	beneficiary  *User
+	contributors []*Contributor
+}
+
+func NewCampaign(title string, description string, goal uint, begin time.Time, deadline time.Time, owner *User, beneficiary *User) *Campaign {
+	return &Campaign{
+		title:        title,
+		description:  description,
+		goal:         goal,
+		current:      0,
+		begin:        begin,
+		deadline:     deadline,
+		owner:        owner,
```

Comment:
Won't be the owner the same as the caller? If not, shouldn't we add an extra requirement to avoid people requesting funds in the name of others?

Reply:
you can see [here](https://miro.com/app/board/uXjVKIt6YJc=/?share_link_id=15852466028) the different types of campaigns I found on the net. So for some campaigns, someone car create a campaign for another person or organism. But I still need to figure out how do some verification on that cases.

Reply:
I agree with you about campaigns for 3p, but you should have a mechanism to avoid people to abuse about the image of others to exploit a campaign.